src/components/layout/layout.js

```diff
@@ -159,7 +161,7 @@ const Layout = ({
         const website_status = useWebsiteStatusApi()
 
         React.useEffect(() => {
-            if (website_status) {
+            if (website_status && !isTestlink()) {
```

Comment:
If you add this here, It will not take the user to endpoint page. Let's keep this file untouched.